src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -972,11 +971,9 @@ uct_dc_mlx5_ep_fc_hard_req_send(uct_dc_mlx5_ep_t *ep,
     UCT_DC_MLX5_IFACE_TXQP_GET(iface, ep, txqp, txwq);
 
     sender.ep                = (uint64_t)ep;
-    sender.payload.seq       = iface->tx.fc_seq++;
+    sender.payload.seq       = fc_entry->seq;
```

Comment:
pls pass just "seq" without fc_entry. we need to make show the function does not change fc hash state

Reply:
sorry, I missed this...
done